apps/mobile/src/components/page/page.layout.tsx

```diff
@@ -18,6 +20,15 @@ export function PageLayout({ children }: HasChildren) {
   const actionBarRef = useRef<ActionBarMethods>(null);
   const contentOffsetBottom = bottom + ACTION_BAR_TOTAL_HEIGHT;
   const contentOffsetTop = top;
+  const [refreshing, setRefreshing] = useState(false);
+
+  const onRefresh = useCallback(() => {
+    setRefreshing(true);
+    queryClient.invalidateQueries();
```

Comment:
Perhaps I should be more nuanced here and invalidate specific balance queries only?

This is a sledgehammer approach to invalidate all queries but we only use wallet/ balance related ones now so I believe thats OK.

Reply:
Yeah, seems ok for now and we can refine it later.